### PR TITLE
Add overwrite status to HDF5 files

### PR DESCRIPTION
**Pull request by rks171**
This is to address Phase 1 of Issue 74.  I ran the MPACT and CTF Nightly's and all continue to pass.  This allows Filetype_HDF5 to be used for reading and writing all CTF HDF5 files.

### CODE REVIEW

File: Futility-CopyrightAndLicense.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2016, Oak Ridge National Laboratories
```
Comment (bscollin) [inline Futility-CopyrightAndLicense.txt]: What is this and how'd it get here?

File: header [hunk 2]
```diff
@@ -0,0 +1,9 @@
+!+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
```
Comment (bscollin) [inline header]: This too

**Comment by bscollin** [comment]
So it looks like you started from casl-dev... which is an issue.  We should just cherry-pick your existing commit onto the head of github master.  (then add the test)

**Comment by rks171** [comment]
I cloned this repo and put the changes in there.  Pushed to a new branch called issue_74_full, which includes the unit test.

**Pull request closed without merging**